<?xml version="1.0" encoding="utf-8"?>

<launch>
    <rosparam command="load" file="$(find anafi_setup)/config/anafi_config.yaml"/>

    <node name="gnss_throttler" type="throttle" pkg="topic_tools" args="messages /anafi/gnss_ned_in_body_frame 2 /anafi/gnss_ned_in_body_frame/downsampled" />
    <node name="image_throttler" type="throttle" pkg="topic_tools" args="messages /anafi/image 10 /anafi/image/downsampled" />

    <node
        name="perception_ekf"
        pkg="perception_ekf"
        type="run_ekf.py"
        output="screen">
        <param name="config_file" value="ekf_config.yaml" />
    </node>
</launch>